### PR TITLE
This is the Security Tooling WG update for the TAC meeting Oct 29.

### PR DESCRIPTION
**Pull request by ware**
This contains consolidated input from all of the ST WG subgroups.

### CODE REVIEW

**Review by steiza — APPROVED** [review]
Thanks!

**Review by marcelamelara — APPROVED** [review]
Thanks @ware ! Exciting work happening in the ST:WG! Does the WG or any of its projects have any questions/asks from the TAC?

**Comment by ware** [comment]
> Thanks @ware ! Exciting work happening in the ST:WG! Does the WG or any of its projects have any questions/asks from the TAC?

No questions or asks at this time.  I probably should have left that section on with a "none".

**Review by sevansdell — APPROVED** [review]
@ware do you feel the TI needs any TAC support? If so, what?

Also, what about Vuln-Reach application status please. https://github.com/ossf/tac/pull/388

File: TI-reports/2024/2024-Q4-ST-WG.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+- Has been approved by both the TAC and LF legal
+- Repo has been moved to a separate GitHub organization
+- Currently using Minder to [enforce branch and CODEOWNER review rules](https://github.com/mindersec/community/tree/main/policies) on the `mindersec` repo
+- Commitment from team to work with Scorecard and Allstar on areas of overlap
+
+## Fuzzing Collaboration
+Chair: Jonathan Metzman
+
+The OSS Fuzzing SIG (Fuzzing Collaboration) continues it's work of supporting open source developers with their fuzzing needs.  There's been no significant changes in this group but they continue to help open source developers to onboard solutions such as OSS-Fuzz into their environments.
```
Comment (lehors) [inline TI-reports/2024/2024-Q4-ST-WG.md:67]:
```suggestion
The OSS Fuzzing SIG (Fuzzing Collaboration) continues its work of supporting open source developers with their fuzzing needs.  There's been no significant changes in this group but they continue to help open source developers to onboard solutions such as OSS-Fuzz into their environments.
```

File: TI-reports/2024/2024-Q4-ST-WG.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- Very successful rollout at CISA's [SBOM-a-Rama 2024](https://www.cisa.gov/news-events/events/sbom-rama-fall-2024)
+- Collaborating with Protobom to enable back-end storage
+- Defining proposal around [SBOM document linking scenarios](https://docs.google.com/document/d/1Dj-OAycyAH3d6A9vPJWldNoLRArRVB607to_0s5Fk8w/edit?tab=t.0#heading=h.qflhf8nb1xeo)
+- Enabled Minder for entire bomctl GitHub org
+
+## Minder
+Co-Chair: Evan Anderson
+Co-Chair: Juan Antonio Osorio
+
+[Minder](https://github.com/mindersec/minder) is an open source platform allowing project owners to proactively manage the security posture several aspects of their SDLC, including their repos, by providing pluggable and customizable checks and policies to minimize risk along their software supply chain and to attest their security practices to downstream consumers.
```
Comment (lehors) [inline TI-reports/2024/2024-Q4-ST-WG.md:57]:
```suggestion
[Minder](https://github.com/mindersec/minder) is an open source platform allowing project owners to proactively manage the security posture of several aspects of their SDLC, including their repos, by providing pluggable and customizable checks and policies to minimize risk along their software supply chain and to attest their security practices to downstream consumers.
```

**Review by lehors — APPROVED** [review]

**Review by bobcallaway — APPROVED** [review]

**Pull request merged**